examples/browser/coverage-webpack.config.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+// @ts-check
+const path = require('path');
```

Comment:
what is the difference with wepback.config.js? can we avoid duplications? can we extract the common part and use object spreading to mix in?

Reply:
The:
.js loader is now
```
 {
+                test: /\.js$/,
+                use: { loader: 'istanbul-instrumenter-loader' },
+                exclude: /node_modules|\.spec\.js$/,
+            },
```

This instruments the code with istanbul.

It may be possible to extend indeed, I guess I have to go and replace one rule in the exports.

I'll do that.